book_my_ticket/sources/book_my_ticket.move

```diff
@@ -27,10 +29,17 @@ module TicketProject::BookMyTicket {
     /// Insufficient amount error code        
     const EINSUFFICIENT_AMOUNT: u64 = 6;
     /// Invalid signature error code          
-    const EINVALID_SIGNATURE: u64 = 7;        
-    
+    const EINVALID_SIGNATURE: u64 = 7;
+    const EUSER_ALREADY_BLACKLISTED : u64 = 8;        
+    const EUSER_NOT_BLOCKED : u64 = 9;
 
     //structs
+    struct OwnerCap has key,store{
+        id : UID ,
+        // listed_admins : VecMap<address,bool>
+
```

Comment:
unused spaces and commented code are here, if it's not required then you can remove it.